src/modules/Channel/components/MessageList/index.tsx

```diff
@@ -173,7 +176,22 @@ export const MessageList = ({
   if (allMessagesFiltered.length < 1) {
     return renderPlaceholderEmpty();
   }
+  const handleUnreadCountClick = () => {
+    if (scrollRef?.current) scrollRef.current.scrollTop = scrollRef.current.scrollHeight;
```

Comment:
This functionality doesn't exist else where? Feels like we're re-inventing the wheel here.

Reply:
This is uikit code that I'm extracting to be re-used in the render prop. I don't think we should refactor their code beyond whats necessary